ups_modul/inout/inout.py

```diff
@@ -1,12 +1,14 @@
-from anthill import *
+from anthill import anthill, anthill_card, anthill_name, anthill_list
 from inout_gsheet import *
```

Comment:
what do you import from inout_gsheet?

Reply:
try_this_month, update_worksheet, user_sheet_log